network-api/networkapi/buyersguide/templates/category_page.html

```diff
@@ -1,8 +1,14 @@
 {% extends "./catalog.html" %}
-{% load static i18n %}
+{% load static i18n env %}
 
 {% block body_id %}{{ category.slug }}{% endblock %}
 
+
+{% block og_image %}
+  <meta property="og:image" content="{% if category.og_image %}{{mediaUrl}}{{"AWS_LOCATION"|env}}/{{category.og_image}}{% else %}{% static "_images/buyers-guide/evergreen-social.png" %}{% endif %}">
```

Comment:
this returns 404. e.g., on review app i get https://res.cloudinary.com/mozilla-foundation-staging/image/upload/network/images/buyersguide/pets_1605297399.png